compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaMemberTypeSymbol.java

```diff
@@ -32,19 +32,19 @@
 import java.util.Optional;
 
 /**
- * Represents a tuple member symbol.
+ * Represents a member type symbol.
```

Comment:
Although we're changing the name, this doesn't just represent the type symbol right? Should we keep the existing description?

Reply:
Let's change it to `member symbol`. I think it doesn't have to be mentioned as tuple or not. WDYT @MaryamZi ?

Reply:
Since this is only relevant to tuples, I think it's OK too say "tuple member".

Reply:
mm. My concern is wont we introduce any other new node to use with this? Doesn't it have to be used abstractly?

Reply:
Can you give an example for a new node that will be relevant to this? Do you mean some other construct which also has members?

Reply:
Yeah. Some new construct.

Reply:
Anyway, I'm +1 for the suggestion. Just wanted to verify

Reply:
Given that the language is stable, IMO it is fairly unlikely to change that way. If there is a change though, we can always update this, given that we'll need significant changes (and it will most likely be a breaking change) anyway.